purchases/src/main/kotlin/com/revenuecat/purchases/Purchases.kt

```diff
@@ -278,6 +280,86 @@ class Purchases @VisibleForTesting(otherwise = VisibleForTesting.PRIVATE) intern
         )
     }
 
+    /**
+     * This method will send a purchase to the RevenueCat backend. Call this when using your own implementation
```

Comment:
```suggestion
     * This method will send a purchase to the RevenueCat backend. Call this when observerMode is set and you're using your own implementation
```

Reply:
hmm or idk, i just think this sentence could be confusing. lots of conditions to be met for you to use this function... let's think about it

Reply:
how about "Call this to sync Amazon purchases after a successful purchase in observerMode or when migrating users to RevenueCat."